App/components/DatabaseComponent.js

```diff
@@ -58,9 +58,8 @@ export default class DataBaseComponent extends React.Component {
 }
 
 // this is suppose to sorts the date in a decending order but it can delete
-const sortByDate = () => {
+const deleteFromDB = () => {
   db.transaction((tx) => {
-    //tx.executeSql("SELECT * FROM items ORDER BY price");
     tx.executeSql("DELETE FROM items");
```

Comment:
So tx.executeSql is what actually performs the action to delete from the db?

Reply:
no, delete from items deletes the whole table.